packages/pyright-internal/src/analyzer/docStringConversion.ts

```diff
@@ -589,7 +589,7 @@ class DocStringConverter {
 
         // catch-all for styles except reST
         const hasArguments =
-            !line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\w+(\s*\(.*?\))*\s*:\s*\w+/g);
+            !line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\w+(\s*\(.*?\))*\s*:\s*[\u4e00-\u9fa5\w+]*/g);
```

Comment:
Did some reading on this and I think adding the `u` flag combined with `\p{L}` is the proper way to handle Unicode letters. This accepts both of the following strings from the issue:
- `        b (str): a中文`
- `        b (str): 中文`

https://www.regular-expressions.info/unicode.html
https://stackoverflow.com/questions/280712/how-can-i-use-unicode-aware-regular-expressions-in-javascript

```suggestion
            !line?.endsWith(':') && !line?.endsWith('::') && !!line.match(/^\s*.*?\w+(\s*\(.*?\))*\s*:\s*\p{L}+/gu);
```